phoenix-hive-base/src/main/java/org/apache/phoenix/hive/util/PhoenixStorageHandlerUtilBase.java

```diff
@@ -20,18 +20,27 @@
 import java.io.ByteArrayInputStream;
```

Comment:
Please mark the *Base classes abstract, to make sure no-one tries to instantiate them.